BPA/Ch.06/Mt.stan

```diff
@@ -49,7 +49,7 @@ generated quantities {
   int<lower=C> N;
   real pr;        // prob never captured given present
   real omega_nd;  // prob present given never captured; same for all animals
-  
+
```

Comment:
You can use compound declare/define for all these, which really makes the programs a lot simpler.